tools/run_tests/run_xds_tests.py

```diff
@@ -1782,6 +1815,90 @@ def _delay(pct):
         set_validate_for_proxyless(gcp, True)
 
 
+def test_csds(gcp, original_backend_service, instance_group, server_uri):
+    logger.info('Running csds')
```

Comment:
`Running test_csds` to match the other test cases.

Reply:
Done.